sdk/storage/Azure.Storage.Queues.Cryptography/CHANGELOG.md

```diff
@@ -0,0 +1,6 @@
+# Release History
+
+## 12.0.0-preview.5
```

Comment:
This whole project was deleted.  Maybe these came from a very old stash?  All Queues.Crypto files should be removed.

Reply:
Ok, Done